build.sbt

```diff
@@ -22,6 +23,12 @@ def common: Seq[Setting[_]] =
     Compile / javacOptions ++= Seq("-encoding", "UTF-8", "--release", "11"),
     Compile / scalacOptions ++= Seq("-encoding", "UTF-8", "-release", "11"))
 
+lazy val codegenProtoAnnotations = project 
+  .in(file("codegen/proto-annotations"))
+  .enablePlugins(PublicProtocolProject)
+  .settings(common)
+  .settings(name := "akkaserverless-codegen-proto-annotations")
```

Comment:
If we want to keep the annotations in the SDK, we need it in an apart sbt module like this because we need it in two palaces. 

We need the codegenCore to compile it and run the introspections tests and we need it to be available for the use code. This is done by adding the annotations as a dependency on the SDK artifacts.